Gemfile

```diff
@@ -61,6 +61,7 @@ group :development, :test do
   # Install a pre-commit hook to enforce code checks before commits
   gem 'mock_redis'
   gem 'pre-commit', '~> 0.38'
+  gem 'rails-controller-testing'
```

Comment:
@trystant Didn't we remove this for a specific reason?

Reply:
We did because we wanted to replace controller tests with request tests. I'm using this in a view spec; I can replace that with a request spec.

Reply:
It seems like you did go ahead and replace that controller spec with a request spec, so can you remove this gem?

Reply:
Yes, I have done this.